packages/react/src/lib/renderComponent.tsx

```diff
@@ -168,7 +170,9 @@ const renderComponent = <P extends {}>(
     props.variables,
   )(siteVariables)
 
-  const animationCSSProp = props.animation ? createAnimationStyles(props.animation, theme) : {}
+  const animationCSSProp = props.animation
+    ? createAnimationStyles(props.animation, context.theme)
```

Comment:
```suggestion
    ? createAnimationStyles(props.animation, theme)
```

 nit: We already destcructed it :)

Reply:
The destructuring was not correct, not sure how to properly destructure it with the correct defaults. Will use only context.theme in the file.

Reply:
Got your point, let's try to refactor it somehow. Now it's super confusing:

![image](https://user-images.githubusercontent.com/14183168/58789953-9c737100-85ef-11e9-8274-197991732aee.png)

Reply:
I am using the context.theme everywhere now, removed the theme const. Please take another look.